dannce/engine/serve_data_DANNCE.py

```diff
@@ -88,7 +88,7 @@ def prepare_data(
             data = np.concatenate((data, dcom), axis=-1)
         elif com_flag:
             # Convert to COM only if not already
-            if len(data.shape) == 3:
+            if len(data.shape) == 3 and CONFIG_PARAMS["n_instances"] == 1:
                 if nanflag:
                     data = np.mean(data, axis=2)
                 else:
```

Comment:
Its because the labelfile for multi instance com is currently a nframe x 3 x ninstances vector. 

generators are handling splitting this up into channels or squashing them into one channel depending on n_channels_out https://github.com/spoonsso/dannce/blob/5e1c2d448c09c6210c3c914700564532ee624c21/dannce/engine/generator_aux.py#L536